### PR TITLE
Change logs level

### PR DESCRIPTION
**Pull request by netanelC**
Change log level

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: src/flow/models/flowsManager.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export class FlowsManager {
     this.logger.debug({ msg: 'got a request for a new flow', flowPayload: payload });
     const url = this.config.get<string>('flowUrl');
     const response = await axios.post<IIngestionResponse>(`${url}/ingestion`, payload);
-    this.logger.info({ msg: 'sent to store-trigger successfully', jobId: response.data.jobID });
+    this.logger.info({ msg: 'sent to store-trigger successfully', jobId: response.data.jobID, payload });
```
Comment (RonitKissis) [inline src/flow/models/flowsManager.ts:15]: add model id

File: src/model/controllers/modelsController.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export class ModelsController {
       return res.status(httpStatus.CREATED).json(createdModel);
     } catch (error) {
       if (error instanceof AppError) {
-        this.logger.error({ msg: `Failed in ingesting a new model! Reason: ${error.message}` });
+        this.logger.error({ msg: `Failed in ingesting a new model! Reason: ${error.message}`, modelPath: req.body.modelPath });
```
Comment (RonitKissis) [inline src/model/controllers/modelsController.ts:23]: add model id

File: src/validator/models/validationManager.ts [hunk 3]
```diff
@@ -164,7 +165,7 @@ export class ValidationManager {
 
     // Intersection validation is only required if productType is "3DPhotoRealistic"
     if (payload.metadata.productType != ProductType.PHOTO_REALISTIC_3D) {
-      this.logger.info({ msg: 'product type is not 3DPhotoRealistic. skipping intersection validation' });
+      this.logger.warn({ msg: 'product type is not 3DPhotoRealistic. skipping intersection validation' });
```
Comment (RonitKissis) [inline src/validator/models/validationManager.ts:168]: add model id

File: src/flow/models/flowsManager.ts (hunk 1 above)
Comment (netanelC) [reply src/flow/models/flowsManager.ts]: Done

File: src/model/controllers/modelsController.ts (hunk 2 above)
Comment (netanelC) [reply src/model/controllers/modelsController.ts:23]: There is no modelId in this block

File: src/validator/models/validationManager.ts (hunk 3 above)
Comment (netanelC) [reply src/validator/models/validationManager.ts]: There is no modelId in this block, I added modelPath

File: src/model/controllers/modelsController.ts [hunk 4]
```diff
@@ -20,7 +20,7 @@ export class ModelsController {
       return res.status(httpStatus.CREATED).json(createdModel);
     } catch (error) {
       if (error instanceof AppError) {
-        this.logger.error({ msg: `Failed in ingesting a new model! Reason: ${error.message}` });
+        this.logger.error({ msg: `Failed in ingesting a new model! Reason: ${error.message}`, modelName: req.body.metadata.productName });
```
Comment (RonitKissis) [inline src/model/controllers/modelsController.ts:23]: model id?

File: src/validator/models/validationManager.ts [hunk 5]
```diff
@@ -72,27 +72,36 @@ export class ValidationManager {
         return 'Bad tileset format. Should be in 3DTiles format';
       }
 
-      this.logger.debug({ msg: 'extracted successfully polygon of the model', polygon: model });
+      this.logger.debug({ msg: 'extracted successfully polygon of the model', polygon: model, modelName: payload.metadata.productName });
 
       const intersection: Feature<Polygon | MultiPolygon> | null = intersect(footprint, model);
 
-      this.logger.debug({ msg: 'intersected successfully between footprint and polygon of the model', intersection });
+      this.logger.debug({
```
Comment (RonitKissis) [inline src/validator/models/validationManager.ts:79]: model id?

File: src/model/controllers/modelsController.ts (hunk 4 above)
Comment (netanelC) [reply src/model/controllers/modelsController.ts:23]: There is no model ID in here

File: src/validator/models/validationManager.ts (hunk 5 above)
Comment (netanelC) [reply src/validator/models/validationManager.ts:79]: There is no model ID here

**Review by RonitKissis — APPROVED** [review]

**Pull request merged**